lib/manageiq/rpm_build/generate_ansible_venv.rb

```diff
@@ -46,7 +46,12 @@ def populate
       def install_python_system_packages
         where_am_i
 
-        shell_cmd("#{pip_versioned} install --no-compile virtualenv ansible-runner")
+        shell_cmd("#{pip_versioned} install --no-compile virtualenv")
+
+        # Install ansible-core dependencies which aren't auto-installed with ansible-core
+        shell_cmd("#{pip_versioned} install --no-compile 'jinja2>=3.0.0' 'resolvelib<0.9.0,>=0.5.3' packaging")
```

Comment:
Doesn't look like we need these.  packagind and resolvelib are already installed as system packages and jinja doesn't appear to be needed